README.adoc

```diff
@@ -28,6 +28,9 @@ Add the upstream wb projects too, because you might want to make changes on thos
 
 * Git clone: `guvnor`, `kie-wb-common`, `drools-wb` too (besides `optaplanner-wb`)
 * Import them all in the *same* IntelliJ project
+* Make sure to use `Maven -> Reimport All Maven Projects` and `Maven -> Generate Sources and Update Folders for All Projects` to update module dependencies
+* Run `grep -r -h --include=beans.xml "exclude" .` in `guvnor`, `kie-wb-common`, `drools-wb` project root directories and paste the output to `optaplanner-wb/optaplanner-wb-webapp/src/main/webapp/WEB-INF/beans.xml`
```

Comment:
Can we not simply make the change in that beans.xml file and commit it, instead of document it.
Especially the fact that we 'd have local, uncommit changes on files that are normally commit is not good.